introduction.md

```diff
@@ -0,0 +1 @@
+Hi everyone! My name is jnguyen7410 and I'll be doing the Mastering Github course!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.